packages/app/src/specs/SpecsListBanners.vue

```diff
@@ -272,6 +276,46 @@ watch(
   { immediate: true },
 )
 
+const bannerCohortOptions = {
+  [BannerIds.ACI_082022_LOGIN]: [
+    { cohort: 'A', value: t('specPage.banners.login.contentA') },
+    { cohort: 'B', value: t('specPage.banners.login.contentB') },
+  ],
+  [BannerIds.ACI_082022_CREATE_ORG]: [
+    { cohort: 'A', value: t('specPage.banners.createOrganization.titleA') },
+    { cohort: 'B', value: t('specPage.banners.createOrganization.titleB') },
+  ],
+  [BannerIds.ACI_082022_CONNECT_PROJECT]: [
+    { cohort: 'A', value: t('specPage.banners.connectProject.contentA') },
+    { cohort: 'B', value: t('specPage.banners.connectProject.contentB') },
+  ],
+}
+
+const getCohortForBanner = (bannerId: string) => {
+  const cohortConfig: CohortConfig = {
+    name: bannerId,
+    options: bannerCohortOptions[bannerId],
+  }
+
+  return useCohorts(cohortConfig)
```

Comment:
After I login, I'm getting this error that freezes the app:
<img width="556" alt="Screen Shot 2022-09-15 at 9 43 26 AM" src="https://user-images.githubusercontent.com/25158820/190434248-ec5cda7f-4be5-4473-8e76-a754a937fcf0.png">

Looks like it's due to the recent refactor. Does this `useCohorts` have to be a composable or can it be converted to be a simple utility function?

Reply:
Argh.  I will take a look

Reply:
Fix committed and pushed.  Working on a test now to validate the fix.